UWPRibbon/App.xaml

```diff
@@ -3,16 +3,13 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:local="using:UWPRibbon"
-    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
-    xmlns:primitiveContract7Present="using:Microsoft.UI.Xaml.Controls.Primitives?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)">
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls">
     <Application.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
-                <!-- Other merged dictionaries here -->
+                <ResourceDictionary Source="Microsoft.Toolkit.Uwp.UI.Controls.Ribbon/Themes/Generic.xaml" />
```

Comment:
What does this do?

Reply:
Merges the styles from the controls' styles into the app resources so I can debug them directly. It's just a diagnostic technique.